public/docs/ts/latest/tutorial/toh-pt1.jade

```diff
@@ -32,164 +33,143 @@ include ../_quickstart_repo
+    Now update the template in the `@Component` decoration with data bindings to these new properties.
 
   +makeExample('toh-1/ts-snippets/app.component.snippets.pt1.ts', 'show-hero')
 
   :marked
-    The browser should refresh and display our title and hero.
+    The browser refreshes and display the title and hero name.
 
-    The double curly braces tell our app to read the `title` and `hero` properties from the component and render them.
+    The double curly braces instruct the app to read the `title` and `hero` properties from the component and render them.
     This is the "interpolation" form of one-way data binding.
 .l-sub-section
   :marked
-    Learn more about interpolation in the [Displaying Data chapter](../guide/displaying-data.html).
+    Learn more about interpolation in the [Displaying Data page](../guide/displaying-data.html).
 :marked
   ### Hero object
 
-  At the moment, our hero is just a name.  Our hero needs more properties.
+  Our hero needs more properties.
   Let's convert the `hero` from a literal string to a class.
 
   Create a `Hero` class with `id` and `name` properties.
-  For now put this near the top of the `app.component.ts` file, just below the import statement.
+  Add these properties near the top of the `app.component.ts` file, just below the import statement.
 
 +makeExample('toh-1/ts/app/app.component.ts', 'hero-class-1', 'app.component.ts (Hero class)')(format=".")
 
 :marked
-  Now that we have a `Hero` class, let’s refactor our component’s `hero` property to be of type `Hero`.
-  Then initialize it with an id of `1` and the name, "Windstorm".
+  Now in your `Hero` class, refactor the component's `hero` property to be of type `Hero`,
+  then initialize it with an id of `1` and the name "Windstorm."
 
 +makeExample('toh-1/ts/app/app.component.ts', 'hero-property-1', 'app.component.ts (hero property)')(format=".")
 
 :marked
-  Because we changed the hero from a string to an object,
-  we update the binding in the template to refer to the hero’s `name` property.
+  Because you changed the hero from a string to an object,
+  update the binding in the template to refer to the hero's `name` property.
 
 +makeExample('toh-1/ts-snippets/app.component.snippets.pt1.ts', 'show-hero-2')
 :marked
-  The browser refreshes and continues to display our hero’s name.
-
-  ### Adding more HTML
-  Displaying a name is good, but we want to see all of our hero’s properties.
-  We’ll add a `<div>` for our hero’s `id` property and another `<div>` for our hero’s `name`.
-
-+makeExample('toh-1/ts-snippets/app.component.snippets.pt1.ts', 'show-hero-properties')
-:marked
-  Uh oh, our template string is getting long. We better take care of that to avoid the risk of making a typo in the template.
+  The browser refreshes and continues to display the hero's name.
 
-  ### Multi-line template strings
+  ### Adding HTML with multi-line template strings
```

Comment:
What's your thought process on the headlines? (Maybe I've missed something - are we standardizing the grammar on them?)